### PR TITLE
Change flags and bundle tag checkout

### PR DESCRIPTION
**Pull request by Gabib32**
Pluralize "bundle" and only pass in the config and tag string.
Use the config to get the bundleUrl information.

Signed-off-by: gabrielle beyer <gabrielle.n.beyer@intel.com>

### CODE REVIEW

File: cmd/check.go [hunk 1]
```diff
@@ -67,7 +67,7 @@ func runBloatCheck(r *diva.Results, u diva.UInfo, args []string) error {
 		u.Ver = args[0]
 	}
 
-	err = diva.GetBundleAtTag(conf, allFlags.bundleURL, u.Ver)
+	err = diva.GetBundlesAtTag(conf, u.Ver)
```
Comment (matthewrsj) [inline cmd/check.go]: need to set `conf.BundleDefsURL` to `allFlags.bundleURL` before calling this function.

File: cmd/pydeps.go [hunk 2]
```diff
@@ -138,7 +138,7 @@ func createFullChroot(path string) error {
 	if pipFlags.version == "0" {
 		err = diva.GetLatestBundles(conf, "")
 	} else {
-		err = diva.GetBundleAtTag(conf, "", pipFlags.version)
+		err = diva.GetBundlesAtTag(conf, pipFlags.version)
```
Comment (matthewrsj) [inline cmd/pydeps.go]: need to set `conf.BundleDefsURL` to `allFlags.bundleURL` before calling this function (I think, if it is available).

File: cmd/check.go (hunk 1 above)
Comment (Gabib32) [reply cmd/check.go]: Do we really want to modify the config struct within the code?

Comment (matthewrsj) [reply cmd/check.go]: yeah, commandline arguments are just an override of the config so that's the closest way to handle it.

Comment (Gabib32) [reply cmd/check.go]: okay dokay

File: cmd/pydeps.go (hunk 2 above)
Comment (Gabib32) [reply cmd/pydeps.go]: I don't pass bundleDefsURL option to this command, but I"ll change that.

**Pull request merged**